cycledash/static/js/examine/components/VCFTable.js

```diff
@@ -3,7 +3,6 @@
 var _ = require('underscore'),
     utils = require('../utils'),
     React = require('react/addons'),
-    $ = require('jquery'),
```

Comment:
Could you try restoring this line and doing something similar to [this sequence][1] in `QueryBox`? Using the NPM jQuery here would fix the tests.

[1]: https://github.com/hammerlab/cycledash/blob/063b06edc946e6c16a80d59a8bebc9ae2b00eaa0/cycledash/static/js/examine/components/QueryBox.js#L16-L21